neuralmonkey/decoders/decoder.py

```diff
@@ -4,6 +4,7 @@
 import tensorflow as tf
 import numpy as np
 
+from neuralmonkey.vocabulary import START_TOKEN
```

Comment:
Představoval jsem si to spíš tak, že by sám objekt Vocabulary měl properties `start_token`, `start_token_index` atd. A pokud by to byla nějaká divná vocabulary (třeba pro klasifikaci), tak by na tyhlety properties házela výjimku. Ale tohle je taky dobrý.